pkg/python/packaging/parse.go

```diff
@@ -26,11 +27,26 @@ func (*Parser) Parse(r dio.ReadSeekerAt) ([]types.Library, []types.Dependency, e
 		return nil, nil, xerrors.Errorf("read MIME error: %w", err)
 	}
 
+	license := h.Get("License-Expression")
+	if license == "" {
+		license = h.Get("License")
+	}
+	if license == "" {
+		for _, classifier := range h.Values("Classifier") {
+			if strings.HasPrefix(classifier, "License :: ") {
+				values := strings.Split(classifier, " :: ")
+				license = values[len(values)-1]
+				break
+			}
+		}
+	}
```

Comment:
We may want to flatten the conditions.

```suggestion
	// "License-Expression" takes precedence as "License" is deprecated.
	// cf. https://peps.python.org/pep-0639/#deprecate-license-field
	var license string
	if l := h.Get("License-Expression"); l != "" {
		license = l
	else if l := h.Get("License"); l != "" {
		license = l
	else {
		for _, classifier := range h.Values("Classifier") {
			if strings.HasPrefix(classifier, "License :: ") {
				values := strings.Split(classifier, " :: ")
				license = values[len(values)-1]
				break
			}
		}
	}
```

Reply:
ok, sure